mender-dist-packages-building/Dockerfile

```
@@ -5,14 +5,21 @@ FROM $DISTRO:$VERSION
 RUN export DEBIAN_FRONTEND=noninteractive && \
     apt-get update && \
     apt-get install -y \
-    build-essential dh-make \
-    git wget curl \
-    debhelper devscripts
+    build-essential \
+    dh-make \
+    git \
+    wget \
+    curl \
+    debhelper \
+    devscripts \
+    equivs
 
 ARG ARCH=amd64
 ARG DISTRO=debian
 ARG VERSION=buster
 
+ENV ARMV6_TOOLCHAIN_ROOT="/opt/armv6-eabihf--glibc--bleeding-edge-2018.11-1"
```

We should stick to the toolchain that we were using. Is there any reason to _downgrade_ it?
```suggestion
ENV ARMV6_TOOLCHAIN_ROOT="/opt/armv6-eabihf--glibc--stable-2020.08-1"
```

---

Discussed during standup, the older toolchain has to stay